client/extensions/woocommerce/state/sites/orders/README.md

```diff
@@ -0,0 +1,63 @@
+Orders
+======
+
+This module is used to manage orders for a site.
+
+## Actions
+
+### `fetchOrders( siteId: number )`
```

Comment:
One more update: This doc needs the `page` argument.

Otherwise, I think everything looks good, and I've began using this pattern for products.